src/core/thread/osthread.d

```diff
@@ -3359,6 +3361,8 @@ package(core.thread) void* getStackTop() nothrow @nogc
 
 package(core.thread) void* getStackBottom() nothrow @nogc
 {
+
+
```

Comment:
?

Reply:
Fixed, that was a leftover from me discovering that you can assign versions inside a function.